sdk/core/Azure.Core.Amqp/src/Azure.Core.Amqp.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>Azure AMQP types</Description>
+    <AssemblyTitle>Microsoft Azure AMQP</AssemblyTitle>
+    <Version>1.0.0-beta.1</Version>
+    <PackageTags>Microsoft Azure AMQP</PackageTags>
+    <Nullable>enable</Nullable>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <NoWarn>$(NoWarn);AZC0001;AZC0012</NoWarn>
```

Comment:
Why did you need the AZC0012 suppression?

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/16341